packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/cli/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,17 @@
+{
+  "pci_ai_dashboard_cli": "Command Line Interface (CLI)",
```

Comment:
**nitpick:** This can be maintained as a constant.

Reply:
Changed here: https://github.com/ovh/manager/pull/8213/commits/9f4aaae0907eaded7c1a7374a222516dd3eb6044